File: DSA_SEM3/Q3.java
#include <stdio.h>
#include <stdlib.h>

int arr[100];

int cmpfunc(const void *a, const void *b) {
    return ((int)a - (int)b);
}

void find_phone_number(int n) {
    int i, j;
    int count = 0;
    for (i = 0; i < n; i++) {
        for (j = 0; j < i; j++) {
            if (arr[i] == arr[j]) {
                break;
            }
        }
        if (i == j) {
            arr[count++] = arr[i];
        }
    }

    for (i = 0; i < count; i++) {
        printf("%d ", arr[i]);
    }
    printf("\n");
}

int main() {
    int n, i;
    scanf("%d", &n);
    for (i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
    find_phone_number(n);
    return 0;
}

File: sem2/q3.cpp
//frequency 
#include<stdio.h>
int main() {
    int a[100],l=0,k=1;

    printf("Enter the no.of elements of the array:");
    scanf("%d",&l);
    
    printf("Enter the elements: \n");
    for(int i=0;i<l;i++) {
        scanf("%d",&a[i]);
    }

    for (int j = 0; j < l ; j++) {
    
            if (a[j] == a[j+1]) {
                k++;
                printf("Frequency of Array[%d] : %d\n",j,k);
            }
            else 
            {
                k=1;
                printf("Frequency of Array[%d] : %d\n",j,k);
            }
    }

    return 0;
}